Planes/MilitaryPlane.js

```diff
@@ -1,4 +1,4 @@
-const Plane = require('./Plane');
+const {Plane} = require('./Plane');
 
 class MilitaryPlane extends Plane {
```

Comment:
Why didn't you add object as a parameter to a `constructor` in `MilitaryPlane`, `ExperimentalPlane`, `Plane` classes?

Reply:
objects were added